content/en/real_user_monitoring/browser/optimizing_performance.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Optimizing Performance
+aliases:
+  - real_user_monitoring/browser/monitoring_performance_vitals/
+further_reading:
+  - link: "https://learn.datadoghq.com/courses/core-web-vitals-lab"
+    tag: "Learning Center"
+    text: "Interactive Lab: Core Web Vitals"
+  - link: "https://www.datadoghq.com/blog/real-user-monitoring-with-datadog/"
+    tag: "Blog"
+    text: "Real User Monitoring"
+  - link: "https://www.datadoghq.com/blog/core-web-vitals-monitoring-datadog-rum-synthetics/"
+    tag: "Blog"
+    text: "Monitor Core Web Vitals with Datadog RUM and Synthetic Monitoring"
+---
+
+## Overview
+
+{{< img src="real_user_monitoring/browser/optimizing_performance/optimization-workflow.mp4" alt="RUM Performance Optimization helps you find the root cause of browser performance issues based on real user traffic." video="true" >}}
+
+The Optimization page helps identify the root cause of browser performance issues using real user traffic data. Here, you can troubleshoot browser metrics such as [Core Web Vitals][1] (CWV), [Loading Time][2] to determine the causes of slow pages, with all relevant information in one place.
```

Comment:
```suggestion
The Optimization page helps to identify the root cause of browser performance issues using real user traffic data. Troubleshoot the causes of slow pages using browser metrics such as [Core Web Vitals][1] (CWV) and [Loading Time][2].
```

Reply:
Really just a suggested revision.

The commas in your second sentence could be removed for clarity, though:
- `[Core Web Vitals][1] (CWV), [Loading Time][2]` could just have an "and" between them.
- `, with all relevant information in one place` isn't really a subordinate clause. It just kind of adds that the info is relevant. It can be removed or added to a new sentence, imo.